frontend/deploy-smoke.js

```diff
@@ -33,12 +32,12 @@ driver
     return value;
   })
   .then((value) => {
-    if (value.includes("success")) {
+    if (value.includes("App status returned success")) {
```

Comment:
does this remove the Okta check from our post-deploy smoke test? I can't remember if we'd decided whether we want to do that

Reply:
Right, I didn't want to add that in since I don't think we ever explicitly called that we wanted to add that to our alerting flow. Even if we did I think we'd probably want a separate alert for it with a specific message, so I opted not to alter the logic for this smoke test accordingly